x-pack/libbeat/management/blacklist.go

```diff
@@ -61,7 +60,7 @@ func NewConfigBlacklist(cfg ConfigBlacklistSettings) (*ConfigBlacklist, error) {
 }
 
 // Detect an error if any of the given config blocks is blacklisted
-func (c *ConfigBlacklist) Detect(configBlocks api.ConfigBlocks) Errors {
+func (c *ConfigBlacklist) Detect(configBlocks ConfigBlocks) Errors {
```

Comment:
@ph @blakerouse Can we remove the filtering for output types (and others) from libbeat as well? These have been introduced in the context of Central Management, in order to disallow some potential risky setting.

Agent should disallow the `console` and `file` input? Filebeat modules should not be configurable anyways. For customizations we already have capabilities in Agent.
Do I miss something why we should keep 'blacklisting' functionality in libbeat?

Reply:
I agree we can remove it, but lets keep the PR focus on removing BCM we can do a follow-up to remove the "reject/accept" functionality after we have this merge. 

Note this old feature is now unnecessary to keep because of the `capabilities` of Elastic Agent.